pioneer/packages/apps-routing/src/tokenomics.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright 2017-2019 @polkadot/apps-routing authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { Routes } from './types';
+
+import Tokenomics from '@polkadot/app-tokenomics/';
+
+export default ([
+  {
+    Component: Tokenomics,
+    display: {
+      needsAccounts: true,
+      needsApi: [
+        'tx.balances.transfer'
```

Comment:
Here you would usually put the name some method that is actually used by the package (ie. `query.stake.stakes`, but this doesn't need to be very strict.